python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -105,6 +106,11 @@ class LoadReferenceObjectsConfig(pexConfig.Config):
         itemtype=str,
         default={},
     )
+    requireProperMotion = pexConfig.Field(
+        doc="Require that the proper motion correction be successful?",
```

Comment:
What does "be successful" mean in this context? The effect of this config parameter is not clear to me.

Reply:
I agree. It means that the data needed to apply proper motion corrections is present. I'll try to improve the help string.